packages/zod-openapi/src/index.ts

```diff
@@ -277,18 +278,24 @@ export class OpenAPIHono<
     return document
   }
 
-  doc = (path: string, config: OpenAPIObjectConfig) => {
-    this.get(path, (c) => {
+  doc = <P extends string>(
+    path: P,
+    config: OpenAPIObjectConfig
+  ): OpenAPIHono<E, S & ToSchema<'get', P, {}, OpenAPIObject>, BasePath> => {
```

Comment:
The below is OK.

```ts
OpenAPIHono<E, S & ToSchema<'get', P, {}, {}>, BasePath>
```

Reply:
@yusukebe 
Since this method does not use the jsonT method, it looks like your suggestion is more correct.
Please check the following commit for the fix.
https://github.com/honojs/middleware/pull/260/commits/3915c72217bae435fdae3335ca1a496bc28370d6

Reply:
> Since this method does not use the jsonT method

Yes, you are right. It does not use `jsonT()`! Your fix is good.